### ISSUE TITLE
asmFishW_2018-05-06_popcnt caused an excpetion during 16min match

### ISSUE THREAD

**Issue by bitsocket** [lantonov/asmFish#167]
During a game the engine suddenly threw an exception and hence the game was terminated.
Fritz-GUI remained stable, engine could just be re-started afterwards, but of course the match was lost.

I guess it might have todo with the Large Pages option, which I had enabled with 6144MB of memory.
The engine had already died once before, when I had increased LP from 4096 to 6144MB.

SYZYGY (5men) was active during the games.

For details see screenshot:
![asmfishw-2018-05-06_exception_this_is_a_problem_in_the_engine](https://user-images.githubusercontent.com/22038957/39865517-6f766876-544d-11e8-965f-bfc71bd6f98b.png)

**Comment by CounterPly** [comment]
Hi bitsocket,

From which source was this _popcnt executable assembled? Does the crash also occur with this pre-assembled binary?

https://github.com/lantonov/asmFish/raw/master/WindowsOS_binaries/asmFishW_2018-04-25_popcnt.exe

**Comment by bitsocket** [comment]
It was assembled on the same machine (Xeon X56) - on which it crashed - twice in a while on 16 min matches only. 

I haven't tried yet to reproduce it with precompiled binaries, as it doesn't crash every now and then. 
The first crash appeared after I had increased hash memory from 4GB to 6GB, while  LP option was still active and SYZYGY was configured in Fritz 16 GUI - not directly with the engine, because F16 GUI doesn't  show the syzygy path option anymore.

Since I have reduced hash mem. size to 4GB again and disabled LP option, it did not crash again - so far.

I will give it a try with precompiled binary and same settings..

**Comment by CounterPly** [comment]
Sorry, I meant from which source code? I just wanted to make sure that it was after this commit: https://github.com/lantonov/asmFish/commit/f9b379372194f0c61177c19befebe769ffe92327. 

This fixed the Large Pages issues encountered by many users.

**Comment by bitsocket** [comment]
It was built from this commit: f9b379372194f0c61177c19befebe769ffe92327, which, according to log messages, already contains the LP fix.

Now, I remember that I had built it from source, AFTER it had thrown the exception twice from the pre-build binary. 

I thought it could be related to different hardware being used for bre-build binaries - compared to my older Xeon X56 LGA1366. Hence, I retried with binaries built on my X56 machine, which however made no difference with regard to the error.
